### PR TITLE
proposal for pip install poetry

### PR DESCRIPTION
**Pull request by j-moreno-c-r**
the idea is create a python venv and inside import the poetry, in this way we avoid conflict problems enter the local pip and the poetry and the pythons...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,7 +69,21 @@ The Krux code is a Python package that should be installed with [Poetry](https:/
 pip install poetry
 poetry install
 ```
-
+There may be a problem at this stage when trying to directly install Poetry, we have a solution proposal(this comand is for unix system mac or linux):
```
Comment (tadeubas) [inline README.md:72]: I would only simplify this line to:
If you have a problem installing Poetry on Linux OS try:

Comment (j-moreno-c-r) [reply README.md:72]: Ok good idea

Comment (j-moreno-c-r) [reply README.md]: better?

Comment (tadeubas) [reply README.md]: awesome! Now just wait for @odudex to merge the PR 😉

**Comment by tadeubas** [comment]
@j-moreno-c-r "This branch has conflicts that must be resolved", you need to update your branch, I the README a short time ago and as you've changed it too it is caused a conflict to git. You'll have to resolve the conflict locally and then update your branch, then the PR here will be automatically updated

**Pull request closed without merging**